src/main/java/gyro/google/compute/InstanceCustomerEncryptionKey.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import com.google.api.services.compute.model.CustomerEncryptionKey;
+import gyro.core.resource.Diffable;
+import gyro.google.Copyable;
+
+public class InstanceCustomerEncryptionKey extends Diffable implements Copyable<CustomerEncryptionKey> {
```

Comment:
This class isn't necessary, use the `gyro.google.compute.EncryptionKey` instead.

Reply:
Done.